features/product/managing_product_reviews/filtering_product_reviews.feature

```diff
@@ -0,0 +1,20 @@
+@managing_product_reviews
+Feature: Browsing product reviews
+    In order to work with multiple reviews
+    As an Administrator
+    I want to filter product reviews
+
+    Background:
+        Given the store has customer "Mike Ross" with email "ross@teammike.com"
+        And the store has a product "Lamborghini Gallardo Model"
+        And this product has a review titled "Awesome" and rated 5 with a comment "Nice product" added by customer "ross@teammike.com"
+        And this product has a new review titled "Bad" and rated 1 added by customer "ross@teammike.com"
```

Comment:
I'm lacking information about the state of these reviews or case in which review is rejected